test/integration/scheduler/framework_test.go

```diff
@@ -139,10 +140,26 @@ func TestReservePlugin(t *testing.T) {
 	// Create a plugin registry for testing. Register only a reserve plugin.
 	registry := framework.Registry{reservePluginName: NewReservePlugin}
 
+	// Setup initial reserve plugin for testing.
+	reservePlugin := schedulerconfig.Plugins{
+		Reserve: []schedulerconfig.Plugin{
+			{
+				Name: reservePluginName,
+			},
+		},
+	}
+	// Set reserve plugin config for testing
+	reservePluginConfig := []schedulerconfig.PluginConfig{
```

Comment:
I think this can be omitted since the reserve plugin doesn't read args.

Reply:
Updated. PTAL.
Thanks!!